src/main/java/com/RestApiTask/RestApiDemo/Controller/MovieNameGetController.java

```diff
@@ -1,15 +1,17 @@
 package com.RestApiTask.RestApiDemo.Controller;
 
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
+import java.util.Map;
 
 @RestController
 public class MovieNameGetController {
 
     @GetMapping("/movie-info")
-    public List<String> getMovieInfo(){
-        return List.of("Dark Waters","Frances Ha");
+    public Map<String,String> getMovieInfo(@RequestParam(name = "movie-list",required = true)String getInfo){
```

Comment:
```suggestion
    public Map<String,String> getMovieInfo(@RequestParam(name = "movie-list",required = true)String movieList){
```
あるいはmoviesとかでもいいかなと思います！

Reply:
https://github.com/sow0731/homework-l7/blob/05cf2995533eb6af8a7e3c3a2cd0241f6ac68ed6/src/main/java/com/rest/api/task/rest/api/demo/controller/MovieNameGetController.java#L14

戻り値を受け取る変数名を"movies"に変更